web_translate_dialog/README.rst

```diff
@@ -1,35 +1,69 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+====================
 Web Translate Dialog
 ====================
 
-This module replaces the standard translation view by an easy-to-use pop-up view where you can translate all the fields of the object in all the installed languages (for the ancient users, it may remind them the translation pop-up of OpenERP 6.1).
+This module replaces the standard translation view by an easy-to-use pop-up
+view where you can translate all the fields of the object in all the installed
+languages (for long-time users, it may remind them the translation pop-up of
+OpenERP 6.1).
+
+This module also features:
 
-This module also features :
-* Adds a *Translate* button item in the *More* menu
-* Support HTML fields
+* A *Translate* button item in the *More* menu
+* Support for HTML fields
 * Autosize the textareas to the size of the content
 
 Usage
 =====
 
-Go to an object that has translatable fields (*Products* for example) and select *More > Translate* (or click on *Edit* and then click on the flag at the top-right of one of the translatable fields): the translation view will pop-up on your screen. This translation view contains all the translatable fields of the object.
+Go to an object that has translatable fields (*Products* for example) and
+select *More > Translate* (or click on *Edit* and then click on the flag at
+the top-right of one of the translatable fields): the translation view will
+pop-up on your screen. This translation view contains all the translatable
```

Comment:
Change this to indicate that clicking on the flag, only the clicked field will be presented.